### PR TITLE
WIP Configure backend to talk to the mobile client

### PR DESCRIPTION
**Pull request by AmberWilkie**
PT Story: https://www.pivotaltracker.com/story/show/133004177

Changes proposed in this pull request:
- Add ionic server `:8000` as an accepted source in `test`

What I have learned working on this feature: 
- Most of what I was doing on this one ended up in the Ionic app - everything except the one line to the accepted URLs, actually. But it's been a fun ride with this `ActionCable` business.

Ready for review!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,6 @@
 
 /coverage
 
-.idea/
\ No newline at end of file
+.idea/
```
Comment (tochman) [inline .gitignore]: About time... 😉

File: app/assets/javascripts/channels/mobile.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        } else {
+            return this.collection().html(data);
+        }
+    },
+    submitAnswer: function (message) {
+        return this.perform('submit_answer', {
+            message: message
+        });
+    }
+});
```
Comment (tochman) [inline app/assets/javascripts/channels/mobile.js]: This feels very repetitive

**Comment by tochman** [comment]
You should try to test these actions with request specs. I think...

File: .gitignore (hunk 1 above)
Comment (diraulo) [inline .gitignore]: It's always been there, @tochman. The diff here is the new line after 🙂

**Pull request closed without merging**